README.md

```diff
@@ -29,28 +29,28 @@ Create a new Telegram bot using [BotFather](https://t.me/BotFather) and obtain a
 ### SSL certificate
 `production only`
 
-Issue a new Let's Encrypt SSL certificate for the domain name using [Certbot](https://certbot.eff.org/)
-Make sure that 80 port is available before running the Certbot.
+Make sure that port 80 is available.\
+Run this command to issue a new SSL certificate for your domain name via Certbot:\
+(NO need to follow official instructions from [Certbot website](https://certbot.eff.org/))
 
 ```bash
 sudo certbot certonly --standalone -d botapi.example.com
 ```
 
-Make sure auto update is enabled and working properly.
+Make sure auto update of your certificate is enabled and working properly.
 
 ### Nginx
 `production only`
 
-Seal app uses Nginx as a reverse proxy. You need to install and configure Nginx to make it work.
-
-Copy `config` file to `/etc/nginx/sites-available`
-
-Edit `config` and set proper values:
+Seal app uses Nginx as a reverse proxy.\
+Install Nginx on your machine (if that's not done already).\
+Copy `config` file from `nginx` foder in your project repo to `/etc/nginx/sites-available` on your machine.\
+Edit this `config` file and set proper values:
```

Comment:
Rename the config file from `/etc/nginx/sites-available/config` to, for example `/etc/nginx/sites-available/botapi.example.com`

Edit config and set proper values:

```bash
sudo nano /etc/nginx/sites-available/botapi.example.com
```